### ISSUE TITLE
Invisible spawn target

### ISSUE THREAD

**Issue by alexcroox** [astruyk/Ares#194]
It would be great to be able to spawn an invisible object that is just a spawning placeholder of sorts. This object can be moved, height changed etc, then we can drag normal zeus objects/units/vehicles onto this object and they will spawn in it's position.

The thinking behind this is it's sometimes frustrating to place items at certain heights and in awkward positions when the typical zeus spawn behaviour is to spawn at ground level. This will allow us to spawn planes on the carrier for example without awkwardly dragging one from nearby land.

**Comment by xgp0006** [comment]
I'll say this indeed would be a big game changer.

**Comment by TheOtherTaylor** [comment]
For my understanding, when you select a unit from the unit list and click it on top of this invisible module, you want the unit to spawn at the module location instead of the default under-cursor spawning?

**Comment by alexcroox** [comment]
Yes in the same way you'd drag a unit into a vehicle "handler" to get them to immediately move in, I imagine an invisible object with the same circle handler, and dragging objects/units onto it's circle handler will spawn them in that position (whatever height it is at).

**Comment by alexcroox** [comment]
What are your thoughts on this @astruyk?

**Comment by astruyk** [comment]
I'm not really sure of the utility. Sure, there's a few special-case times it'd be useful, but it's not really something that seems to have generic appeal. I'm not sure I'd ever have used it.

**Comment by alexcroox** [comment]
Have you ever tried to spawn units inside a building that players are currently clearing? Not ideal I know, but often I'll want to spawn units in a building I didn't realise they'd spend time clearing, so I'll spawn a couple of guys on the top floor, but the default zeus behaviour might spawn them on the ground floor right infront of them before I get a chance to drag them to the top floor!
